src/pageObject.js

```diff
@@ -1,17 +1,40 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
-
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
 };
 
 class ArticlePage extends PageObject {
-
+  constructor(url, slug) {
+    super(url);
+    this.url = url + slug;
 };
 
+get commentButton() {
+  return '[Publish comment] button';
+}
+
+clickOnCommentButton() {
+  return 'Click on the ' + this.commentButton;
+}
+
+assertPageOpened() {
+  return 'The ' + this.url + ' is opened';
```

Comment:
There is a minor formatting issue in the `assertPageOpened` method. The returned string should have a space before 'is opened'. It should be 'The ' + this.url + ' is opened'.